### PR TITLE
Add build-time support for customizable default API and URL endpoints

### PR DESCRIPTION
**Pull request by kyparisisg**
**Description:**
This update allows the use of build-time values to customize the default API and URL endpoints.

**Usage:**
You can override the default endpoints during the build process by using the following -ldflags options during `go build`:
```
-ldflags "-X main.defaultAPI=https://your-custom-api.com -X main.defaultURL=https://your-custom-url.com"
```

### CODE REVIEW

File: cmd/yopass/main.go [hunk 1]
```diff
@@ -37,10 +37,16 @@ Examples:
...
+var (
+	defaultAPI = "https://api.yopass.se"
+	defaultURL = "https://yopass.se"
+)
+
 func init() {
-	// Defaults
-	viper.SetDefault("api", "https://api.yopass.se")
-	viper.SetDefault("url", "https://yopass.se")
+    // Use build-time values if set; otherwise, fall back to hardcoded defaults.
```
Comment (jhaals) [inline cmd/yopass/main.go:46]: could you make sure this comment is correctly indented :)

**Review by jhaals — CHANGES_REQUESTED** [review]
Great addition! Just a minor thing and then good to 🚢

File: cmd/yopass/main.go (hunk 1 above)
Comment (kyparisisg) [reply cmd/yopass/main.go]: It should be fixed now :)

**Review by jhaals — APPROVED** [review]
Thanks!

**Pull request merged**